### PR TITLE
Remove MSG from SignalR

### PR DESCRIPTION
**Pull request by anurse**
This **includes** the changes from #2005, until I merge that (at which point I will rebase).

### CODE REVIEW

**Comment by anurse** [comment]
Changing base to make @BrennanConroy happy. I'll change back to dev after #2005 is in.

**Comment by anurse** [comment]
Rebased on dev after #2005

**Comment by JamesNK** [comment]
Are there any docs that mention a Msg pack package that need to be changed?

**Comment by anurse** [comment]
Nope: https://github.com/aspnet/Docs/search?utf8=%E2%9C%93&q=MsgPack&type=

File: clients/ts/FunctionalTests/package-lock.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@types/debug": {
       "version": "0.0.30",
       "resolved": "https://registry.npmjs.org/@types/debug/-/debug-0.0.30.tgz",
-      "integrity": "sha512-orGL5LXERPYsLov6CWs3Fh6203+dXzJkR7OnddIr2514Hsecwc8xRpzCapshBbKFImCsvS/mk6+FWiN5LyZJAQ==",
+      "integrity": "sha1-3B5A9687nIFQE6eGDmJS9jUqhN8=",
```
Comment (BrennanConroy) [inline clients/ts/FunctionalTests/package-lock.json]: revert!

File: src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack/MsgPackProtocolDependencyInjectionExtensions.cs [hunk 2]
```diff
@@ -1,31 +1,30 @@
-using System;
-using System.Linq;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.AspNetCore.SignalR.Protocol;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using System;
```
[the remark this answers is not in the document]
Comment (anurse) [reply src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack/MsgPackProtocolDependencyInjectionExtensions.cs]: My VS settings flipped somehow :(

**Pull request merged**